Assets/Scripts/Game/Questing/Actions/SkillLevel.cs

```diff
@@ -0,0 +1,94 @@
+// Project:         Daggerfall Tools For Unity
+// Copyright:       Copyright (C) 2009-2018 Daggerfall Workshop
+// Web Site:        http://www.dfworkshop.net
+// License:         MIT License (http://www.opensource.org/licenses/mit-license.php)
+// Source Code:     https://github.com/Interkarma/daggerfall-unity
+// Original Author: Hazelnut
+// Contributors:    
+// 
+// Notes:
+//
+
+using System.Text.RegularExpressions;
+using FullSerializer;
+using DaggerfallConnect;
+using System;
+
+namespace DaggerfallWorkshop.Game.Questing
+{
+    /// <summary>
+    /// Triggers when player has reached specified skill level or higher.
+    /// </summary>
+    public class SkillLevel : ActionTemplate
+    {
+        DFCareer.Skills skill;
+        int minSkillValue;
+
+        public override string Pattern
+        {
+            get { return @"skill (?<skillName>\w+) (?<minSkillValue>\d+)"; }
```

Comment:
Just a minor thing, but would you mind changing this to "when skill" so it reads more like a trigger condition?

If you want to go full throttle, you could make it "when skill (?<skillName>\w+) is at least (?<minSkillValue>\d+)" haha :)